.travis.yml

```diff
@@ -6,8 +6,21 @@ cache:
   directories:
    - $HOME/.m2
 
-# skip install
-install: true
+install:
```

Comment:
can we cache some of these steps? Or is the download install update pretty fast?